ui/components/MesheryMeshInterface/PatternService/helper.js

```diff
@@ -76,6 +76,70 @@ function getXKubenetesToRJSFCompatibleFieldType(schema) {
   return returnedType || false;
 }
 
+// Order of properties in the form
+const sortOrder = [
+  "string",
+  "integer",
+  "number",
+  "boolean",
+  "array",
+  "object"
+];
+
+/**
+ * Sorts the properties of the jsonSchema in the order of the sortOrder.
+ * @param {*} properties
+ * @param {*} sortOrder
+ * @returns
+ */
+export const sortProperties = (properties) => {
```

Comment:
isn't it taking only one param? In the code where you're managing `allOf`, `oneOf` etc and in the JSdoc, you have used it as if it could take two.

Reply:
yeah, it take one param, something residual.